### DISCUSSION TITLE
Quick installation test: ${SITE} list

### DISCUSSION THREAD

**Discussion by criticalcake** [umputun/remark42#863] [Q&A] [answered]
The documentation states:

> To verify if remark has been properly installed, check a demo page at ${REMARK_URL}/web URL. Make sure to include remark site id to ${SITE} list.

I was wondering, what `${SITE}` list means:
Is this just a single value (or multi values) for the SITE parameter?
I do not have to declare/modify a ${SITE} variable somewhere?

**Comment by akellbl4** [comment, marked as answer]
By default `SITE` is `remark`.
This parameter describes IDs of sites that use Remark42. It is used to connect your fronted with backend.
For example, if you use one instance of Remark42 with more than one site, you can specify list of sites and then connect each site to their own site id by set `site_id` in `remark_config` when setting it up in HTML.
You can find the list of all the parameters here: https://github.com/umputun/remark42#parameters

**Reply by criticalcake** [reply]
ok, thank you. That's what I expected. My HTML uses the same site_id.
